src/ecalc/libraries/libecalc/common/libecalc/input/validation_errors.py

```diff
@@ -116,6 +117,7 @@ def __init__(
     ):
         errors = validation_error.errors()
 
+        component_name = data[EcalcYamlKeywords.name]
```

Comment:
Are we sure that the `EcalcYamlKeywords.name` is always a part of this dict? Just asking... Better to:

```
component_name = data.get(EcalcYamlKeywords.name, "Missing Component Name")
```

?

Reply:
I was thinking the same, but assumed it was safe when all tests passed. Will update.

Reply:
"Missing component name" does not make a lot of sense if the dto validation error is for fuel or facility input. We should probably figure out where to 'catch' these errors, but currently it can be something different than a component.